pkg/controller/machineset/machineset_controller_test.go

```diff
@@ -148,3 +249,21 @@ func cleanResources(namespace string) error {
 
 	return nil
 }
+
+// isRetryableAPIError returns whether an API error is retryable or not.
+// inspired by: k8s.io/kubernetes/test/utils.
+func isRetryableAPIError(err error) bool {
```

Comment:
Was this copied from somewhere else within this repo? Should it be in a utils packaged?